introduction.md

```diff
@@ -7,4 +7,6 @@ I guess to satisfy the challenge, another favorite path is JavaScript
 * JavaScript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?